tests/cover/test_testdecorators.py

```diff
@@ -71,10 +71,10 @@ def is_not_too_large(x):
         if x >= 10:
             raise ValueError('No, %s is just too large. Sorry' % x)
 
-    with raises(ValueError) as exinfo:
+    with raises(ValueError) as e:
         is_not_too_large()
 
-    assert ' 10 ' in exinfo.value.args[0]
+    assert ' 10 ' in e.value.args[0]
```

Comment:
e is used fairly consistently:

```py
src/hypothesis/internal/conjecture/engine.py:        except StopTest as e:
tests/cover/test_conjecture_test_data.py:    with pytest.raises(StopTest) as e:
tests/cover/test_health_checks.py:    with raises(FailedHealthCheck) as e:
tests/cover/test_health_checks.py:    with raises(FailedHealthCheck) as e:
tests/cover/test_health_checks.py:    with raises(FailedHealthCheck) as e:
tests/cover/test_health_checks.py:    with raises(FailedHealthCheck) as e:
tests/cover/test_health_checks.py:    with raises(FailedHealthCheck) as e:
tests/cover/test_internal_helpers.py:    with pytest.raises(TypeError) as e:
tests/cover/test_validation.py:    with pytest.raises(InvalidArgument) as e:
tests/cover/test_validation.py:    with pytest.raises(InvalidArgument) as e:
tests/cover/test_validation.py:    with pytest.raises(InvalidArgument) as e:
tests/cover/test_reflection.py:    with raises(TypeError) as e:
tests/cover/test_reflection.py:    with raises(TypeError) as e:
tests/cover/test_reflection.py:    with raises(TypeError) as e:
tests/cover/test_reflection.py:    with raises(TypeError) as e:
tests/cover/test_testdecorators.py:    with raises(ValueError) as e:
tests/cover/test_testdecorators.py:    with raises(Unsatisfiable) as e:
tests/cover/test_testdecorators.py:    with raises(Unsatisfiable) as e:
tests/cover/test_find.py:    with pytest.raises(NoSuchExample) as e:
tests/cover/test_find.py:    with pytest.raises(NoSuchExample) as e:
tests/cover/test_find.py:    with pytest.raises(NoSuchExample) as e:
tests/cover/test_find.py:    with pytest.raises(Timeout) as e:
tests/nocover/test_statistical_distribution.py:    with pytest.raises(AssertionError) as e:
tests/nocover/test_statistical_distribution.py:    with pytest.raises(ConditionTooHard) as e:
tests/nocover/test_statistical_distribution.py:    with pytest.raises(AssertionError) as e:
```

I'm happy to use something else if you like, but consistency should trump picking a one-off. If it's consistent, then when you decide to change your mind, you won't inconsistently pick.